apps/mobile/store/authStorage.ts

```
@@ -0,0 +1,37 @@
+    AsyncStorage.setItem("user", stringifyJson(user)),
+  ]);
+};
+
+export const clearAuthData = async () => {
+  await Promise.all([AsyncStorage.removeItem("accessToken"), AsyncStorage.removeItem("user")]);
+};
+
+export const handleAuthStorage = async (type: string, data: LoginData) => {
+  switch (type) {
+    case WEBVIEW_MESSAGE_TYPES.SIGN_IN_SUCCESS:
+      const { user, accessToken } = data;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)